IntroToCSharp/Shared/Components/Engage/Questions/MultipleChoiceQuestion/MultipleChoiceQuestion.razor

```diff
@@ -136,6 +136,9 @@
     private string Path => $"{NavigationManager.GetRoute()}/questions/{Question}";
     private DataReference<MultipleChoiceQuestionData>? _dataReference;
 
+    private DateTime _lockTimer;
+    private DateTime _unlockTime;
```

Comment:
Variables are not used. These should probably be part of a different feature branch.

Reply:
This must have been transferred over from something else I was working on. Thanks for pointing that out.